Source/CombatExtended/CombatExtended/Loadouts/Dialog_ManageLoadouts.cs

```diff
@@ -72,7 +72,7 @@ public Dialog_ManageLoadouts(Loadout loadout)
             	CurrentLoadout = loadout;
             SetSource(SourceSelection.Ranged);
             doCloseX = true;
-			doCloseButton = true;
+			//doCloseButton = true; //Close button is awkward 
```

Comment:
Reasoning behind this?

Reply:
Close button draws over item selection row, and awkward